lightly/api/api_workflow_client.py

```diff
@@ -104,10 +104,14 @@ def __init__(
                     )
                 )
         except (
+            # TODO(guarin, 07/23): Why do we have to catch value errors?
             ValueError,
+            # Any error by API client if status not in [200, 299].
             ApiException,
-            LightlyAPITimeoutException,
-            AttributeError,
```

Comment:
Do you remember why we caught the `AttributeError`? If not, I'd keep it in here.

Reply:
Yes, it was because SIGALRM was not supported on windows.